hack/generate/samples/internal/go/memcached-with-webhooks/e2e_test_code.go

```diff
@@ -288,15 +288,15 @@ var _ = Describe("memcached", func() {
 
 			By("validating that pod(s) status.phase=Running")
 			getMemcachedPodStatus := func() error {
-				cmd = exec.Command("kubectl", "get",
-					"pods", "-l", "app=memcached", "-o", "jsonpath={.items[*].status}",
-					"-n", namespace,
+				cmd =  exec.Command("kubectl", "get", "-l",
```

Comment:
@NikhilSharmaWe it seems missing the pod "kubectl", "get", "pods", then we have the label
So, it will probably fail